docs/FAQ/how-to-deploy-bold-bi-and-bold-reports-in-an-ack-cluster.md

```diff
@@ -0,0 +1,337 @@
+
+    ![PV Claim](images/ack_pv_claim.png)
+
+6. If you have a DNS to map with the application, proceed to the next steps, otherwise, skip to **Step 10**. 
+
+7. In common_ack_deploy.yaml file and locate line 2335. Uncomment the host value and replace example.com with your DNS hostname.
+
+      ![DNS](images/ack_ingress_http_yaml.png)
+  
+8. If you have an SSL certificate for your DNS and need to configure the site with it, follow these steps; otherwise, skip to **Step 10**.
+
+9. In the ingress spec of the common_ack_deploy.yaml file, uncomment the tls section. Replace example.com with your DNS hostname and save the file.
+
+      ![ingress DNS](images/ack_ingress_tls_yaml.png)
+
+10. Run the following command to retrieve the External IP address:
+
+      ```sh
+      kubectl get svc -n kube-system
+      ```
+      Repeat the above command until you find the IP address listed in the EXTERNAL-IP column, as shown in the following image:
+
+      ![Ingress Address](images/ack_ingress_address.png)
+
+11. Note the EXTERNAL-IP address and map it to your DNS if you have added it in the common_ack_deploy.yaml file. If you do not have a DNS, use the EXTERNAL-IP address to access the application.
+
+12. Replace `<application_base_url>` with your DNS or EXTERNAL-IP address.
+
+    Examples: `http://example.com`, `https://example.com`, `http://<external_ip_address>`
+    
+    ![App_Base_Url](images/ack_app_base_url.png) 
+    
+13. Read the optional client library license agreement from the following link.
+
+    [Consent to deploy client libraries](../docs/consent-to-deploy-client-libraries.md)
+    
+14. By default, all client libraries for Bold BI and Bold Reports will be installed in Kubernetes. However, you can overwrite them by specifying the required libraries as a comma-separated list in the environment variable noted from the above link.
+
+    ![Optinal_Lib](images/ack_optional_lib.png) 
+
+15. Deploy Bold BI and Bold Reports in your Kubernetes cluster by running the following command:
+
+    ```sh
+    kubectl apply -f common_ack_deploy.yaml
+    ```
+
+16. Create a TLS secret with your SSL certificate using the following command:
+
+    ```sh
+    kubectl create secret tls bold-tls -n bold-services --key <key-path> --cert <certificate-path>
+    ```
+
+17. Check the status of the pods by running the following command:
+
+     ```sh
+    kubectl get pods -n bold-services
+     ```
+    ![Pod status](images/ack_pod_status.png) 
+
+18. Wait until you see the applications in the running state. Then, use your DNS or EXTERNAL-IP address obtained from the below command to access the application in the browser.
+
+    ```sh
+    kubectl get ingress -n bold-services
+     ``` 
+
+19. For more details on configuring the application startup, please refer to the [this](#application-startup)
```

Comment:
19. For more details on configuring the application startup, please refer to [this](#application-startup).